### ISSUE TITLE
How to not get banned

### ISSUE THREAD

**Issue by nadirzebiri** [fawazahmed0/youtube-uploader#216]
Looks like i'm the only one who got banned after using your library?? I followed the instructions well and even tested twice on different accounts.

Luckily, I didnt test your script on my main account

So how to not get banned from using this

**Comment by nadirzebiri** [comment]
I give up, if you want to upload videos in a safer way use the official documentation: https://developers.google.com/youtube/v3/guides/uploading_a_video

**Comment by wanghaisheng** [comment]
please tell me more about your usage

**Comment by nadirzebiri** [comment]
Hi [wanghaisheng](https://github.com/wanghaisheng), I didn't do anything wrong. I just uploaded two youtube shorts and channel got banned

**Comment by wanghaisheng** [comment]
1.do you use a proxy?
2.do you use user/pass login?
3.

**Comment by nadirzebiri** [comment]
1. no (idk, it shouldn't cause any problem)
2. yes

**Comment by nadirzebiri** [comment]
Yep, the accounts are new. Also what do you mean by "how could you use a temp phone to verify?" ?

**Comment by wanghaisheng** [comment]
for new account to upload video,  only if your video is more than 30 minutes, you dont need 2 steps verify/phone verify at all.
I think mostly because of this temp phone service is detected by youtube instead of upload bot

**Comment by wanghaisheng** [comment]
I have 3 account use my own bot and no one got banned at all

**Comment by wanghaisheng** [comment]
@znadir any news

**Comment by znadir** [comment]
Yes, actually i did get banned even without using a temp phone service. I will try using your repo later

**Comment by znadir** [comment]
Thank you I would not have thought that you would come back a month later to ask me for news... xD

**Comment by znadir** [comment]
there is also another reason why I could be banned. Maybe it's because my script was taking tiktok videos to post them on youtube.

**Comment by wanghaisheng** [comment]
>By the way, there is another reason why I could be banned. Maybe it's because my script was taking tiktok videos to post them on Youtube.

it should not be an issue. there are bunch of people do things like that.and if banned for this reason, it will tells you about community policy breaking or something like that.

I was intend to upgrade this one into some new features

https://github.com/wanghaisheng/youtube-uploader-ts2py/blob/main/plans.md

**Comment by pierreminiggio** [comment]
I've 3 Youtube accounts that have been posting using this library, and none got banned.
So there is probably something you do in your process of managing the accounts that makes Youtube think you're a bot.

So you'll have to do more trials and errors to figure out what exactly makes Google flags your accounts.

Just to outline how I roll with them at the moment : My 3 accounts are running on a remote server, and they all connect to a self hosted VPN at my place.

**Comment by wanghaisheng** [comment]
@pierreminiggio 

>a self hosted VPN at my place.

you mean setup a socks proxy using your resident ip for remote server?

**Comment by pierreminiggio** [comment]
Yes it was indeed a socks proxy more specifically.
Sorry for wrong terminology, I woke up not too long ago 😂

**Comment by wanghaisheng** [comment]
thank for sharing

**Comment by helmarluiz** [comment]
@znadir are you still getting banned? 

I've created 4 channels and got all banned. I received the following email from youtube:

"We have reviewed your content and found severe or repeated violations of our Community Guidelines. Because of this, we have removed your channel from YouTube.

How we find violations
--
We use a combination of automated systems and human reviews to detect violations of our Community Guidelines.
 
How this affects your channel
 
We have permanently removed your channel from YouTube. Going forward, you won't be able to access, possess, or create any other YouTube channels.
 

How we find violations
 
We use a combination of [automated systems](https://notifications.google.com/g/p/ANiao5pNrjvxiGGR97BsL6DLvlMg0csh8nqAA0rQYqPsz7G5942eVwHG3ZZwhtVQFe2doMJTMuHU0-gkUzJaYpOQtsygQzMOYwLf88NikU0HCveQAUXeVrnTxZSmgB5hfXHr2KPM0k9OnBuGO6iLc-TT-RYlisG9_LE7ppxlxmNUOeL6WKVHRFU57pFLUfCry3F8JE9sd6oeR_5CnsWPtpDy_Bqt2NPbq5aoZQDXhU_cLA) and human reviews to detect violations of our Community Guidelines.

 
How this affects your channel
 
We have permanently removed your channel from YouTube. Going forward, you won't be able to access, possess, or create any other YouTube channels."

**Comment by znadir** [comment]
@helmarluiz I tried twice uploading youtube video with this repository and got banned twice. I've since given up, I don't want to mess with big sites like YouTube
